index.html

```diff
@@ -4797,7 +4745,7 @@ <h3>Proofs (Signatures)</h3>
         <h3>Expiration</h3>
 
         <p>
-The <code>expirationDate</code> is expected to be within an expected range
+The <code>validUntil</code> is expected to be within an expected range
 for the <a>verifier</a>. For example, a <a>verifier</a> can check that the
```

Comment:
This goes with @dlongley's suggestion (which applies to the next line of the document), and together they make the description of `validUntil` correspond to the description of `validFrom`
```suggestion
for the <a>verifier</a>. For example, a <a>verifier</a> can check that the end of the
```